scripts/js_ts_linter.py

```diff
@@ -0,0 +1,1065 @@
+        between the arguments of the controller function.
+        """
+        if self.verbose_mode_enabled:
+            python_utils.PRINT(
+                'Starting controller dependency line break check')
+            python_utils.PRINT('----------------------------------------')
+        files_to_check = self.all_filepaths
+        failed = False
+        summary_messages = []
+
+        # For RegExp explanation, please see https://regex101.com/r/T85GWZ/2/.
+        pattern_to_match = (
+            r'controller.* \[(?P<stringfied_dependencies>[\S\s]*?)' +
+            r'function\((?P<function_parameters>[\S\s]*?)\)')
+        stdout = python_utils.string_io()
+        with linter_utils.redirect_stdout(stdout):
+            for filepath in files_to_check:
+                file_content = FILE_CACHE.read(filepath)
+                matched_patterns = re.findall(pattern_to_match, file_content)
+                for matched_pattern in matched_patterns:
+                    stringfied_dependencies, function_parameters = (
+                        matched_pattern)
+                    stringfied_dependencies = (
+                        stringfied_dependencies.strip().replace(
+                            '\'', '').replace(' ', ''))[:-1]
+                    function_parameters = (
+                        function_parameters.strip().replace(' ', ''))
+                    if stringfied_dependencies != function_parameters:
+                        failed = True
+                        python_utils.PRINT(
+                            'Please ensure that in file %s the line breaks '
+                            'pattern between the dependencies mentioned as '
+                            'strings:\n[%s]\nand the dependencies mentioned '
+                            'as function parameters: \n(%s)\nfor the '
+                            'corresponding controller should '
+                            'exactly match.' % (
+                                filepath, stringfied_dependencies,
+                                function_parameters))
+                        python_utils.PRINT('')
+
+            if failed:
+                summary_message = (
+                    '%s   Controller dependency line break check failed, '
+                    'see messages above for the affected files.' % (
+                        _MESSAGE_TYPE_FAILED))
+                python_utils.PRINT(summary_message)
+                summary_messages.append(summary_message)
+            else:
+                summary_message = (
+                    '%s  Controller dependency line break check passed' % (
+                        _MESSAGE_TYPE_SUCCESS))
+                python_utils.PRINT(summary_message)
+                summary_messages.append(summary_message)
+
+            python_utils.PRINT('')
+        self.process_manager['line_breaks'] = summary_messages
+        # _STDOUT_LIST.append(stdout)
```

Comment:
Ditto here and every other occurrence of this.

Reply:
Removed.